### ISSUE TITLE
Question about a centralized settings.json

### ISSUE THREAD

**Issue by fruza** [fullphat/snarl#58]
Hi!, when I use a centralized `settings.json` file and I access to the management server through `http://localhost:9989/` and change any options and I clic save. Where is the options saved ?? ... I do not see any change made in the centralized `settings.json` file.

Regards, Fernando.

**Comment by fullphat** [comment]
That's by design, otherwise all users could individually modify a single shared settings file.  The expectation is that the centralised file would be managed manually by an administrator/system owner.

**Comment by fruza** [comment]
Ok, I understand!. I like that behavior however when I change something through management server for one computer I guess it's saved in a temporary place. Where?, can I access to this temp file?. I could be interesting to test some behavior for one computer changing options and then if it's right set those options to the centralized `settings.json` file.

Regards, Fernando.

**Comment by fullphat** [comment]
Yes, if you have a machine running Snarl that doesn't have the admin settings applied, the settings file will be saved in `%APPDATA\full phat\snarl 5.0\etc\settings.json` you can then copy this file to the central location for your managed Snarl instances
